doc/source/package_versions.rst

```diff
@@ -1,14 +1,44 @@
+PyGranta and Granta MI compatibility
```

Comment:
This page is now referred to by the recordlist, jobqueue, and bomanalytics READMEs for further instructions on how to select a compatible version.

Reply:
The title of the page has changed, but the name of the file is the same, so any existing links should still work.